pdfixed/src/pd_helpers.cpp

```diff
@@ -33,13 +33,16 @@ pd_bytes_meter_spec_to_rates(p4_pd_bytes_meter_spec_t *meter_spec) {
 
   // bytes per microsecond
   info_rate = static_cast<double>(meter_spec->cir_kbps) / 8000.;
+  // 10^6 bytes
```

Comment:
that's incorrect
it's converting it to bytes, not Mbytes

Reply:
Oops, you are correct.